public/js/ui.js

```diff
@@ -1,15 +1,32 @@
-
-
-function renderAlertMessage(elemId, text) {
-  var span = document.getElementById(elemId);
-
-  span.appendChild(document.createTextNode(text));
+var handler = undefined;
+function renderAlertMessage(msg, variant, time) {
+
+  if(handler !== undefined) clearAlertMessage(handler);
+  if(msg === undefined) msg = "Something went wrong";
+  if(variant === undefined) variant = "error";
+  if(time === undefined) time = 4000;
+
+  const element = document.getElementById("alertbox");
+  
+  element.classList.add("show", variant);
+  handler = setTimeout(clearAlertMessage, time);
+  
+  element.innerHTML = `
```

Comment:
what is browser support like for template literals - should we transpile this or is it safe in the last three versions of popular browsers?

Reply:
Except for internet explorer, it is safe to use. I have collected the data from https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals
![Screenshot 2019-09-03 at 11 03 05 AM](https://user-images.githubusercontent.com/22586210/64157277-0a7c3e00-ce54-11e9-8c82-dfdd6db98971.png)

Yes, we have to use transpiler, for IE.